en/identity-server/next/mkdocs.yml

```diff
@@ -866,6 +866,11 @@ nav:
       - SCIM 2.0 Bulk API: apis/organization-apis/org-scim2-bulk-mgt.md
       - SCIM 2.0 Group management API: apis/organization-apis/org-group-mgt.md
       - SCIM 2.0 Role management API: apis/organization-apis/org-role-mgt.md
+      - Server management:
```

Comment:
"Server Management" topic is not required 
Let's make Email templates and Notification Templates Management API first class

Reply:
Fixed with https://github.com/wso2/docs-is/pull/5034/commits/b8d4ab4d4cbd1a407fdfcf0ae3287da4b691cad1